### ISSUE TITLE
Add a way to list out trials instances from GH action

### ISSUE THREAD

**Issue by malomarrec** [sourcegraph/sourcegraph#44845]
#### Problem

Today, CEs need to determine, from an account name or email address, what trial instance is associated with the account. Note that instances typically have random subdomains.

Use cases include:
- helping reset passwords
- checking activity on an instance

There is no source of truth that CEs can access to do this. Some information is available in SFDC, but there's no guarantee that is up to date.
Authorized users can get this data by running:

```bash
gcloud projects list --filter='name~sourcegraph-managed-src' --format="json(projectId,labels)"
```

#### Proposal

As a quick fix, until we have a dedicated dashboard/service, add a way to run this in GitHub Actions, and make this available to CEs.

- [x] Add to GitHub action @filiphaftek 
- [x] Document in cloud handbook @filiphaftek 
- [x] Document in cloud operations manual @malomarrec  (and document limitations of SFDC data)
- [x] Send a message in #ce to make sure everybody is aware @malomarrec 
- [x] remove `PLG Cloud Instance` field

#### Out of scope longer term fix

- A dashboard that acts as the source of truth for that
- Having one subscription per trial and automatically linking to SFDC (blocked on being able to programmatically generate subscriptions and licenses)
- One-way, "realtime" sync of this data into SFDC

**Comment by malomarrec** [comment]
https://github.com/sourcegraph/console/pull/211 was merged. All items are closed now. Ongoing conversation about Looker data ingestion in [slack](https://sourcegraph.slack.com/archives/C03B4EMBXL4/p1670017487794719).
Follow up issue

- [ ] Instance - company map in analytics

**Comment by malomarrec** [comment]
Hey @malomarrec, this issue didn't have a team label. Given you created it, I assigned you and added your team label. Feel free to unassign yourself. If this is not the right label, can you make sure to add the right one?
